pkg/kwok/controllers/pod_controller.go

```diff
@@ -420,6 +424,12 @@ func (c *PodController) watchResources(ctx context.Context, opt metav1.ListOptio
 				case watch.Added, watch.Modified:
 					pod := event.Object.(*corev1.Pod)
 					if c.need(pod) {
+						if event.Type == watch.Added && c.nodeMetricsHandler != nil {
+							c.nodeMetricsHandler.AddContainersCount(pod.Spec.NodeName, len(pod.Spec.Containers))
```

Comment:
```suggestion
							c.nodeMetricsHandler.AddContainersCount(pod.Spec.NodeName, 1)
```

Reply:
Wouldn't that count the number of pods, not the number of containers?

Reply:
Yep, this is cumulative number of containers started

Reply:
But if we replace len(pod.Spec.Containers) with 1, and the pod contains 5 containers for example, this would increase the count by 1, while the number of containers is actually 5? Or I completely misunderstood the metric we are trying to apply :smile:

Reply:
``` yaml
help: [ALPHA] Cumulative number of containers started
```

Yes this is correct, only after the kwok-controller is restarted will it start accumulating from 0 again

Reply:
But changing the value from len to 1 would not really solve the restart problem since number of containers is always >= 1. I mean I or you can apply this suggestion if you think this is a right approach, I'm just trying to understand why is the len of pod.Spec.Containers wrong here :relaxed: